extensions/coffee/src/utils.ts

```diff
@@ -32,7 +31,8 @@ function preventArguments(args?: string | undefined) {
 export async function stopCaffeinate(updateMenubar = true, hudMessage?: string) {
   if (updateMenubar) {
     try {
-      await launchCommand({ name: "index", type: LaunchType.Background, context: { caffinated: false } });
+      await launchCommand({ name: "index", type: LaunchType.Background, context: { caffeinated: false } });
+      await launchCommand({ name: "status", type: LaunchType.Background, context: { caffeinated: false } });
```

Comment:
Now, also immediately updates the status of the new `status` command. The `index` command is now solely responsible for the menu bar item